### DISCUSSION TITLE
How to enable typescript inlay hints in svelte

### DISCUSSION THREAD

**Discussion by gustavojoaquin** [jellydn/ts-inlay-hints#52] [Q&A]
### Describe the bug

Hi, I configured typescript-tools nvim and enabled the inlay hints, but can't enable the inlay hints for svelte 

### Reproduction

Install svelte language server in neovim

### System Info

```Shell
Operating system: Arch Linux
Neovim: 0.10
```

### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/antfu/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by jellydn** [comment]
Hi @GustavoJCL That would must be implemented with Svelte LSP to make it works. You should check the document there https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#svelte

**Comment by jellydn** [comment]
FYI - Found this recently https://github.com/MysticalDevil/inlay-hints.nvim/blob/master/README.md#svelte-language-server
